docs/src/index.rst

```diff
@@ -1,19 +1,88 @@
-scikit-matter
-=============
+What is scikit-matter?
+======================
 
 scikit-matter is a toolbox of methods developed in the
 computational chemical and materials science community, following the
 `scikit-learn <https://scikit.org/>`_ API
 and coding guidelines to promote usability and interoperability with existing workflows.
 
+.. raw:: html
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 d-flex">
+                <div class="card text-center intro-card shadow">
+                <div class="card-body flex-fill">
+                    <h5 class="card-title">
+
+
+.. only:: html
+
+  .. image:: /examples/selection/images/thumb/sphx_glr_FeatureSelection-WHODataset_thumb.png
+    :alt:
+
+  :ref:`getting_started-selection`
```

Comment:
This is super nice these cards!

Can you maybe set the background to transparent. Dark mode user here. 😝

Also, can we maybe have the three cards in one line for widescreens and responsivly stacked. Seems like you are using bootstrap. I think there is an option. We can look together.

Reply:
Agreement on the transparency, also a dark mode user

Reply:
added